agents/commands.py

```
@@ -0,0 +1,245 @@
+            log.warning(f"Unknown command: /{command_name}")
+            return {"error": f"Unknown command: /{command_name}"}
+
+        command = self.commands[command_name]
+        return command.handler(args, context or {})
+
+    def list_commands(self, category: Optional[CommandCategory] = None) -> list[dict]:
+        """List available commands"""
+        commands = []
+        seen = set()
+
+        for name, cmd in self.commands.items():
```

Ruff (0.15.14)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)